/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/pandas/core/computation/expressions.py:21: UserWarning: Pandas requires version '2.8.4' or newer of 'numexpr' (version '2.7.3' currently installed).
  from pandas.core.computation.check import NUMEXPR_INSTALLED
11:18 bilby INFO    : Loaded 158 posteriors
Traceback (most recent call last):
  File "/local/condor/execute/dir_3202212/gwpopulation_pipe_analysis", line 8, in <module>
    sys.exit(main())
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 431, in main
    hyper_prior = load_prior(args)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 95, in load_prior
    hyper_prior = ConditionalPriorDict(filename=filename)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/dict.py", line 667, in __init__
    super(ConditionalPriorDict, self).__init__(
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/dict.py", line 44, in __init__
    self.from_file(filename)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/dict.py", line 158, in from_file
    self.from_dictionary(prior)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/dict.py", line 279, in from_dictionary
    dictionary[key] = cls.from_repr(args)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/base.py", line 335, in from_repr
    return cls._from_repr(string)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/base.py", line 349, in _from_repr
    kwargs[key] = cls._parse_argument_string(val)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/bilby/core/prior/base.py", line 444, in _parse_argument_string
    other_cls = getattr(import_module(module), other_cls)
AttributeError: module 'bilby.core.prior' has no attribute '1/'
/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/pandas/core/computation/expressions.py:21: UserWarning: Pandas requires version '2.8.4' or newer of 'numexpr' (version '2.7.3' currently installed).
  from pandas.core.computation.check import NUMEXPR_INSTALLED
11:45 bilby INFO    : Loaded 158 posteriors
Traceback (most recent call last):
  File "/local/condor/execute/dir_3793002/gwpopulation_pipe_analysis", line 8, in <module>
    sys.exit(main())
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 432, in main
    model = load_model(args)
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 138, in load_model
    model = cls([_load_model(model, args) for model in args.models.values()])
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 138, in <listcomp>
    model = cls([_load_model(model, args) for model in args.models.values()])
  File "/home/storm.colloms/.conda/envs/O4apop/lib/python3.10/site-packages/gwpopulation_pipe/data_analysis.py", line 191, in _load_model
    raise ValueError(f"Model {model} not found.")
ValueError: Model variable_m2min not found.
